tests/integration/gov/keeper/grpc_query_test.go

```diff
@@ -89,7 +89,8 @@ func TestGRPCQueryTally(t *testing.T) {
 
 				expRes = &v1.QueryTallyResultResponse{
 					Tally: &v1.TallyResult{
-						YesCount:        math.NewInt(3 * 5 * 1000000).String(),
+						// HV2: weighted operations not allowed
```

Comment:
It looks like the test is for non-weighted votes, we really expect zero votes here ?

Reply:
Great catch!
So my findings:
1. The test is failing because id = 0 in the setup (and this is not allowed). I fixed it, but
2. This keeps failing because of the usage of `IterateCurrentValidatorsAndApplyFn` which returns `nil` here (the real implementation is in our custom `stake` module). 
So I think we need to skip it anyway, but for a different reason.
Updated the comment. Thanks